integration-tests/gradle/src/test/java/io/quarkus/gradle/devmode/JandexMultiModuleProjectDevModeTest.java

```diff
@@ -0,0 +1,34 @@
+package io.quarkus.gradle.devmode;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.google.common.collect.ImmutableMap;
+
+public class JandexMultiModuleProjectDevModeTest extends QuarkusDevGradleTestBase {
+
+    @Override
+    protected String projectDirectoryName() {
+        return "jandex-basic-multi-module-project";
+    }
+
+    @Override
+    protected String[] buildArguments() {
+        return new String[] { "clean", ":application:quarkusDev", "-s" };
+    }
+
+    protected void testDevMode() throws Exception {
+
+        assertThat(getHttpResponse())
+                .contains("ready")
+                .contains("my-quarkus-project")
+                .contains("org.acme.quarkus.sample")
+                .contains("1.0-SNAPSHOT");
```

Comment:
This one is checking the content of the index.html. Which you remove given that you removed the index.xml.

Reply:
Will remove it

Reply:
The idea behind removing this assert statement is to straightaway check for the /hello endpoint right ?

Reply:
The idea is that the application no longer serves the `index.html` and it suffices to just check `/hello` - checking the contents of `index.html` doesn't provide anything useful to this test

Reply:
Okk